test-reqs.txt

```diff
@@ -1,3 +1,5 @@
+nose
```

Comment:
Am I misunderstanding that ``test-reqs.txt`` is the requirements file for testing?